### DISCUSSION TITLE
Apply to 4 octernships limit

### DISCUSSION THREAD

**Discussion by Debangshu97** [community/community#51063] [GitHub Education]
So I have applied to 4 octernships limit. Will I be able to apply to any else in the future? I am confused.
Thank You.

**Comment by Ntshangase** [comment]
Hi, 

Yes, You will be allowed to apply again, as long as you remain a verified student.

**Comment by jaiakash** [comment]
Yeah. You can apply to more than 4 if the deadline of other projects get passed. For eg, I applied 3 projects and the deadline of 2 projects got ended. So now i can apply to 3 more projects (one project still has deadline).
It seems at one time,  you can apply to 4 projects.

**Comment by DevRelMalima** [comment]
Yeah you can apply to more of such project once you student pack has not expired
